### PR TITLE
[build][pytorch][neuron] Upgrade Pillow version in PT Neuron DLC

### PR DESCRIPTION
**Pull request by saimidu**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [x] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [x] (If applicable) I've documented below the tests I've run on the DLC image
- [x] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [x] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

#### EIA/NEURON Checklist
* When creating a PR:
- [x] I've modified `src/config/build_config.py` in my PR branch by setting `ENABLE_EI_MODE = True` or `ENABLE_NEURON_MODE = True`
* When PR is reviewed and ready to be merged:
- [x] I've reverted the code change on the config file mentioned above

## Reviewer Checklist
* For reviewer, before merging, please cross-check:
- [ ] I've verified the code change on the config file mentioned above has already been reverted

*Description:*
Fix safety issues

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron [hunk 1]
```diff
@@ -104,11 +103,11 @@ RUN pip install --no-cache-dir -U \
     grpcio==1.24.1 \
     protobuf==3.10.0 \
     gast==0.2.2 \
-    pillow==7.2.0 \
+    pillow==8.2.0 \
     "awscli<2"
 
 RUN pip install neuron-cc[tensorflow] --extra-index-url https://pip.repos.neuron.amazonaws.com \
- && pip install "torch-neuron>=1.5.1,<1.5.2" --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && pip install "torch-neuron>=1.5.1.1.2,<1.5.1.1.3" --extra-index-url https://pip.repos.neuron.amazonaws.com \
```
Comment (tejaschumbalkar) [inline pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron:110]: why is this added?

File: test/dlc_tests/sanity/test_telemetry.py [hunk 2]
```diff
@@ -27,6 +27,7 @@ def test_telemetry_bad_instance_role_disabled_cpu(cpu, ec2_client, ec2_instance,
 @pytest.mark.processor("neuron")
 @pytest.mark.integration("telemetry")
 @pytest.mark.parametrize("ec2_instance_type", ["inf1.xlarge"], indirect=True)
+@pytest.mark.skip("Feature doesn't exist on Neuron DLCs")
```
Comment (tejaschumbalkar) [inline test/dlc_tests/sanity/test_telemetry.py:30]: is this a placeholder if we are skipping the test for neuron?

File: test/dlc_tests/sanity/test_telemetry.py [hunk 3]
```diff
@@ -51,6 +52,7 @@ def test_telemetry_instance_tag_success_cpu(cpu, ec2_client, ec2_instance, ec2_c
...
```
Comment (tejaschumbalkar) [inline test/dlc_tests/sanity/test_telemetry.py:55]: is this a placeholder if we are skipping the test for neuron?

File: pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron (hunk 1 above)
Comment (saimidu) [reply pytorch/inference/docker/1.5.1/py3/Dockerfile.neuron:110]: There are multiple builds for `torch-neuron`, and all builds which are `1.5.1.1.2.*` are targeted at ubuntu16.04, while all versions `1.5.1.1.3.*` are targeted at ubuntu18.04, which is why I've constrained the version.

File: test/dlc_tests/sanity/test_telemetry.py (hunk 2 above)
Comment (saimidu) [reply test/dlc_tests/sanity/test_telemetry.py:30]: Yes, this is a placeholder for when it is added in to the framework source code.

File: test/dlc_tests/sanity/test_telemetry.py (hunk 3 above)
Comment (saimidu) [reply test/dlc_tests/sanity/test_telemetry.py:55]: Yes, this is a placeholder for when it is added in to the framework source code.

**Comment by tejaschumbalkar** [comment]
Soft Approved. Lets revert the temp configs

**Review by tejaschumbalkar — APPROVED** [review]
